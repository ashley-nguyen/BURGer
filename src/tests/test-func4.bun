def string ret1(string s){
	return s;
	}

string t = ret1("omg");
print(t);

def char ret2(char c){
	return c;
	}
char ch = ret2("h");
print(ch);

def double ret3(double d){
	return d;
	}
double doub = ret3(4.25);
print(doub);

def int ret4(int i){
	return i;
	}
int a = ret4(5);
print(a);
