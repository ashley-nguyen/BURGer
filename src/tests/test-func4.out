omg
h
4.25
5

